.github/workflows/release.yml

```diff
@@ -349,8 +349,9 @@ jobs:
             rerun_io_docs_link="https://rerun.io/docs"
             py_docs_link="https://ref.rerun.io/docs/python/${{ needs.version.outputs.final }}"
           else
-            web_app_link="https://app.rerun.io/commit/${{ needs.version.outputs.release-commit }}"
-            rerun_io_docs_link="https://rerun.io/preview/${{ needs.version.outputs.release-commit }}/docs"
+            short_commit_hash=$(echo ${{ needs.version.outputs.release-commit }} | cut -c1-7)
+            web_app_link="https://app.rerun.io/commit/$short_commit_hash"
```

Comment:
The commit uploaded to gcloud uses the short hash, which broke this link

Reply:
That reminds me: we should make an issue to be consistent with hashes, preferably use the 7-prefix everywhere

Reply:
https://github.com/rerun-io/rerun/issues/4131